testing/xfpga/test_object_c.cpp

```diff
@@ -23,21 +23,25 @@
 // CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
+#include <fstream>
 #include "gtest/gtest.h"
 #include "test_system.h"
+#include "types_int.h"
 #include "xfpga.h"
 
 using namespace opae::testing;
 
+const std::string DATA =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
 class sysobject_p : public ::testing::TestWithParam<std::string> {
  protected:
   sysobject_p() : tmpsysfs_("mocksys-XXXXXX") {}
 
   virtual void SetUp() override {
     ASSERT_TRUE(test_platform::exists(GetParam()));
     ASSERT_EQ(xfpga_fpgaGetProperties(nullptr, &dev_filter_), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetObjectType(dev_filter_, FPGA_DEVICE),
-              FPGA_OK);
+    ASSERT_EQ(fpgaPropertiesSetObjectType(dev_filter_, FPGA_DEVICE), FPGA_OK);
     ASSERT_EQ(xfpga_fpgaGetProperties(nullptr, &acc_filter_), FPGA_OK);
     ASSERT_EQ(fpgaPropertiesSetObjectType(acc_filter_, FPGA_ACCELERATOR),
               FPGA_OK);
```

Comment:
In some of the tests you're call Enumerate, but you're not freeing tokens_ in TearDown.